doc/fix_ablate.txt

```diff
@@ -225,3 +242,8 @@ This fix can only be used in simulations that define implicit surfaces.
 "read isurf"_read_isurf.html
 
 [Default:] none
```

Comment:
Please list the default value for the new keyword, for example see https://github.com/sparta/sparta/blob/master/doc/fix_ave_time.txt#L298-L302

Reply:
Updated documentation to indicate default value